apps/user_status/src/store/userStatus.js

```diff
@@ -252,6 +252,25 @@ const actions = {
 		commit('loadStatusFromServer', status)
 	},
 
+	/**
+	 * Stores the status we got in the reply of the heartbeat
+	 *
+	 * @param {Object} vuex The Vuex destructuring object
+	 * @param {Function} vuex.commit The Vuex commit function
+	 * @param {Object} status The data destructuring object
+	 * @param {String} status.status The status type
+	 * @param {Boolean} status.statusIsUserDefined Whether or not this status is user-defined
+	 * @param {String} status.message The message
+	 * @param {String} status.icon The icon
+	 * @param {Number} status.clearAt When to automatically clear the status
+	 * @param {Boolean} status.messageIsPredefined Whether or not the message is predefined
+	 * @param {string} status.messageId The id of the predefined message
```

Comment:
```suggestion
	 * @param {String} status.messageId The id of the predefined message`
```
Sorry for that nitpick, but I can't unsee that.